scss/_root.scss

```diff
@@ -158,6 +158,9 @@
     color-scheme: dark;
 
     // scss-docs-start root-dark-mode-vars
+    --#{$prefix}success-icon: #{escape-svg($success-icon-dark)}; // Boosted mod
```

Comment:
Can't we have the same approach for light and dark mode? For the light mode, it's handled with `$svg-as-custom-props` with a loop to create `--#{$prefix}#{$icon}-icon`.